src/lib/hooks/useFlashMintTrade.ts

```diff
@@ -39,8 +39,8 @@ export const useFlashMintTrade = () => {
         const { tx } = quote
         const defaultGasEstimate = BigNumber.from(6_000_000)
         const gasEstimatooor = new GasEstimatooor(signer, defaultGasEstimate)
-        // Will throw error if tx would fail
-        // If the user overrides, we take any gas estimate
+        // `estimate` will throw error if tx would fail
+        // If the user overrides, we use the `defaultGasEstimate`
         const canFail = override
```

Comment:
Why don't you just invert the `override` boolean here?

I don't like changing the whole gas estimator defaults just for this particular case.

Reply:
Updated - let's discuss more when you're back 👍